File: sys/js/fs.js
export function cwd() {
  return Deno.cwd()
}

export async function readText(fpath, options={}) {
  return await Deno.readTextFile(fpath)
}

export async function writeText(fpath, text, options={}) {
  return await Deno.writeTextFile(fpath, text)
}

/*
export async function dirList(fpath) {
    let list = []
    for await (const entry of Deno.readDir(fpath)) {
        list.push(entry)
    }
    return list
}
global.dirList = dirList
*/

global.cwd = cwd
global.readText = readText
global.writeText = writeText

File: sys/script1/lib/src/fs.dart
import 'dart:io';

Directory cwd() {
    return Directory.current;
}

Future<String> readText(file) async {
    var f = File(file);
    var text = await (f.readAsString());
    return text;
}

Future<File> writeText(file, text) async {
    var f = File(file);
    return await (f.writeAsString(text));
}
